profiles/source_profile.go

```diff
@@ -261,6 +261,7 @@ type SourceProfileConnectionDynamoDB struct {
 	AwsRegion          string // Same as AWS_REGION environment variable
 	DydbEndpoint       string // Same as DYNAMODB_ENDPOINT_OVERRIDE environment variable
 	SchemaSampleSize   int64  // Number of rows to use for inferring schema (default 100,000)
+	Streaming          string // Confirmation for streaming migration (required yes)
```

Comment:
Can this be `bool`?

Reply:
This Streaming is taken as input in the source-profile flag. The source-profile flag is a string variable. After taking it as input  we parse it into a map with key and value both as string datatype.
> Even if we keep this Streaming as bool then after parsing into string we have to typecast that string into boolean. 

If we are keeping this as a boolean then we will expect customers to respond with "true"  to this parameter. So instead of keeping boolean we can directly keep this as a string and compare it with true.
To make it look more like boolean we can change "yes" to "true".

Reply:
Thanks for the clarification, makes sense!
In this case, I have two suggestions - 
1. We can name the flag to reflect its dichotomous nature, maybe something like `enableStreaming` or `useStreaming`?
2. I have used some CLIs in the past which is accept - (yes/no/true/false) all as valid inputs. Maybe we can replicate that.

Reply:
> We can name the flag to reflect its dichotomous nature, maybe something like enableStreaming or useStreaming?

Yeah you are right! This looks more simple and clear for someone using the feature.

> I have used some CLIs in the past which is accept - (yes/no/true/false) all as valid inputs. Maybe we can replicate that.

We can do that and also add to README about what are the valid options to enter for starting migration.